### DISCUSSION TITLE
peak_ranges

### DISCUSSION THREAD

**Discussion by akhst7** [stuart-lab/signac#1349] [Q&A] [answered]
In a section of Generating a peak/cell or bin/cell matrix under Analyzing PBMC scATAC-seq vignette,  describes how to calculate a peak_matrix as follows; 

```
peak_matrix <- FeatureMatrix
  fragments = obj@fragments,
  features = peak_ranges
)
```
What I am not sure about is the `peak_range`.  The `peak_ranges` is defined as  "a set of genomic ranges spanning the set of peaks to be quantified per cell" but I am still not sure how to derive the `peak_range`.   I'd really appreciate any pointers.

**Comment by timoast** [comment, marked as answer]
Hi, please see the documentation for `FeatureMatrix()`: https://stuartlab.org/signac/reference/featurematrix

One way to generate a set of genomic ranges is to perform peak calling: https://stuartlab.org/signac/articles/peak_calling.html

**Comment by akhst7** [comment]
Thanks.
